resources/js/components/ui/Table/Table.tsx

```diff
@@ -155,28 +167,56 @@ export const Table = <T,>(optProps: Props<T>) => {
                   )}
                 </For>
               </div>
-              <Index
-                each={props.table.getRowModel().rows}
+              <Show
+                when={props.rowsIteration === "For"}
                 fallback={
-                  <div class={ts.wideRow}>
-                    <Show when={props.isDimmed} fallback={EMPTY_VALUE_SYMBOL}>
-                      <Spinner />
-                    </Show>
-                  </div>
+                  <Index
+                    each={props.table.getRowModel().rows}
+                    fallback={
+                      <div class={ts.wideRow}>
+                        <Show when={props.isDimmed} fallback={EMPTY_VALUE_SYMBOL}>
+                          <BigSpinner />
+                        </Show>
+                      </div>
+                    }
+                  >
+                    {(row) => (
+                      <div class={ts.dataRow} inert={props.isDimmed || undefined}>
+                        <Index each={row().getVisibleCells()}>
+                          {(cell) => (
+                            <span class={ts.cell}>
+                              <CellRenderer component={cell().column.columnDef.cell} props={cell().getContext()} />
+                            </span>
+                          )}
+                        </Index>
+                      </div>
+                    )}
+                  </Index>
                 }
               >
-                {(row) => (
-                  <div class={ts.dataRow} inert={props.isDimmed || undefined}>
-                    <Index each={row().getVisibleCells()}>
-                      {(cell) => (
-                        <span class={ts.cell}>
-                          <CellRenderer component={cell().column.columnDef.cell} props={cell().getContext()} />
-                        </span>
-                      )}
-                    </Index>
-                  </div>
-                )}
-              </Index>
+                <For
+                  each={props.table.getRowModel().rows}
+                  fallback={
+                    <div class={ts.wideRow}>
+                      <Show when={props.isDimmed} fallback={EMPTY_VALUE_SYMBOL}>
+                        <BigSpinner />
+                      </Show>
+                    </div>
+                  }
+                >
+                  {(row) => (
+                    <div class={ts.dataRow} inert={props.isDimmed || undefined}>
+                      <Index each={row.getVisibleCells()}>
+                        {(cell) => (
+                          <span class={ts.cell}>
+                            <CellRenderer component={cell().column.columnDef.cell} props={cell().getContext()} />
+                          </span>
+                        )}
+                      </Index>
+                    </div>
+                  )}
+                </For>
+              </Show>
```

Comment:
Takie rozwiązanie zapobiegnie duplikacji:
```tsx
<Dynamic
  component={{Index, For}[props.rowsIteration()]}
  each={props.table.getRowModel().rows}
  fallback={
    <div class={ts.wideRow}>
      <Show when={props.isDimmed} fallback={EMPTY_VALUE_SYMBOL}>
        <BigSpinner />
      </Show>
    </div>
  }
>
  {(row) => (
    <div class={ts.dataRow} inert={props.isDimmed || undefined}>
      <Index each={row.getVisibleCells()}>
        {(cell) => (
          <span class={ts.cell}>
            <CellRenderer component={cell().column.columnDef.cell} props={cell().getContext()} />
          </span>
        )}
      </Index>
    </div>
  )}
</Dynamic>
```

Reply:
No niestety nie bo w jednym row jest akcesorem a w drugim nie jest :) Sam bym tak zrobił gdyby nie to. A tak to jeszcze trzeba by zawartość wyciągnąć gdzieś i tak jakoś nie byłem przekonany że to pomoże.

Reply:
A poprawiłem jednak.